lib/membrane/parent_spec.ex

```diff
@@ -160,6 +160,8 @@ defmodule Membrane.ParentSpec do
     end
   end
 
+  @opaque link_builder_t :: link_builder_t()
```

Comment:
shouldn't that be

```suggestion
  @opaque link_builder_t :: LinkBuilder.t()
```

please check if dialyzer passes